docs/framework/deployment/initialization-errors-managing-the-user-experience.md

```diff
@@ -53,7 +53,7 @@ Displaying an error message to communicate that a requested .NET Framework versi
 
 The [ICLRMetaHostPolicy::GetRequestedRuntime](../unmanaged-api/hosting/iclrmetahostpolicy-getrequestedruntime-method.md) method accepts a [METAHOST_POLICY_FLAGS](../unmanaged-api/hosting/metahost-policy-flags-enumeration.md) enumeration member as input. You can include the METAHOST_POLICY_SHOW_ERROR_DIALOG flag to request an error message if the requested version of the CLR is not found. By default, the error message is not displayed. (The [ICLRMetaHost::GetRuntime](../unmanaged-api/hosting/iclrmetahost-getruntime-method.md) method does not accept this flag, and does not provide any other way to display the error message.)
 
-Windows provides a [SetErrorMode](https://go.microsoft.com/fwlink/p/?LinkID=255242) function that you can use to declare whether you want error messages to be shown as a result of code that runs within your process. You can specify the SEM_FAILCRITICALERRORS flag to prevent the error message from being displayed.
+Windows provides a [SetErrorMode](https://docs.microsoft.com/windows/win32/api/errhandlingapi/nf-errhandlingapi-seterrormode) function that you can use to declare whether you want error messages to be shown as a result of code that runs within your process. You can specify the SEM_FAILCRITICALERRORS flag to prevent the error message from being displayed.
```

Comment:
```suggestion
Windows provides a [SetErrorMode](/windows/win32/api/errhandlingapi/nf-errhandlingapi-seterrormode) function that you can use to declare whether you want error messages to be shown as a result of code that runs within your process. You can specify the SEM_FAILCRITICALERRORS flag to prevent the error message from being displayed.
```